js/main.js

```diff
@@ -21,6 +23,81 @@ function saveEntry(event) {
   data.entries.unshift(entryData);
   $image.setAttribute('src', '/images/placeholder-image-square.jpg');
   $formInputs.reset();
+  data.view = 'entries';
+  $ul.prepend(renderEntry(entryData));
+  $noEntriesText.className = 'hidden';
+  swapViews();
+  $formInputs.reset();
+}
+
+$formInputs.addEventListener('submit', handleSubmit);
+
+function renderEntry(entry) {
+  var $listItem = document.createElement('li');
+  $listItem.setAttribute('data-entry-id', entry.entryId);
+
+  var $row = document.createElement('div');
+  $row.setAttribute('class', 'row margin-bottom50');
+  $listItem.appendChild($row);
+
+  var $colHalfImg = document.createElement('div');
+  $colHalfImg.setAttribute('class', 'column-half center');
+  $row.appendChild($colHalfImg);
+
+  var $img = document.createElement('img');
+  $img.setAttribute('src', entry.photoUrl);
+  $colHalfImg.appendChild($img);
+
+  var $colHalfEntry = document.createElement('div');
+  $colHalfEntry.setAttribute('class', 'column-half');
+  $row.appendChild($colHalfEntry);
+
+  var $entryH2 = document.createElement('h2');
+  $entryH2.textContent = entry.title;
+  $colHalfEntry.appendChild($entryH2);
+
+  var $entryP = document.createElement('p');
+  $entryP.textContent = entry.notes;
+  $colHalfEntry.appendChild($entryP);
+
+  return $listItem;
+}
+
+function handleDomContent(event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    var newEntry = renderEntry(data.entries[i]);
+    $ul.append(newEntry);
+    $noEntriesText.className = 'hidden';
+  }
+}
+window.addEventListener('DOMContentLoaded', handleDomContent);
+
+var $headerEntries = document.querySelector('.header-entries');
+var $newButton = document.getElementById('new-btn');
+var $entryForm = document.querySelector('.view');
+var $entries = document.querySelector('.entries');
+
+function swapViews() {
+  if (data.view === 'entries') {
+    $entries.className = 'entries';
+    $entryForm.className = 'hidden';
+  } else if (data.view === 'entry-form') {
+    $entries.className = 'hidden';
+    $entryForm.className = 'entry-form';
+  }
+}
+
+function swapViewToEntries(event) {
+  data.view = 'entries';
+  swapViews();
+}
+$headerEntries.addEventListener('click', swapViewToEntries);
+
+function swapViewToForm(event) {
+  data.view = 'entry-form';
+  swapViews();
 }
+$newButton.addEventListener('click', swapViewToForm);
 
-$formInputs.addEventListener('submit', saveEntry);
+window.addEventListener('load', swapViews(event));
```

Comment:
this isn't actually doing what you think it is, you can just call `swapViews` in your `DOMContentLoaded` event handler